### DISCUSSION TITLE
java.vendor.url.bug system property

### DISCUSSION THREAD

**Discussion by yamadamn** [microsoft/openjdk#20] [Q&A] [answered]
Hi,

Environment:
```
openjdk version "11.0.10" 2021-01-19
OpenJDK Runtime Environment Microsoft-18724 (build 11.0.10+9)
OpenJDK 64-Bit Server VM Microsoft-18724 (build 11.0.10+9, mixed mode)
```

I've checked the system properties of the preview build.
And, it shows ``java.vendor.url.bug = https://github.com/microsoft/openjdk-releases/issues``.
I can't access the above URL for now.
Is this wrong or be public later?

Regards,

**Comment by sunnychanwork** [comment, marked as answer]
This looks like a bug - the url should be https://github.com/microsoft/openjdk/issues - notice that -release is missing

**Reply by karianna** [reply]
It is indeed a bug, we'll get that sorted for the PSU release coming up ~April 20th.  https://github.com/microsoft/openjdk/issues/21
